src/utils/bootstrapUtils.js

```diff
@@ -126,6 +130,10 @@ export function getClassSet(props) {
   return classes;
 }
 
+export function omitBsProps(props) {
```

Comment:
This only takes care of the basic bs props. I'm curious, are you thinking of expanding on this concept to have something like omitProps(props, tagName) ?